src/components/ActionMenu/ActionMenu.tsx

```diff
@@ -110,17 +90,25 @@ export class ActionMenu extends PureComponent<ActionMenuProps, State> {
         )}
       </div>
     );
-  };
+  }
 
-  private handleMenuGroupToggle = (group: string) => {
-    this.setState(({activeMenuGroup}) => ({
-      activeMenuGroup: activeMenuGroup ? undefined : group,
-    }));
-  };
+  function handleMenuGroupToggle(group: string) {
+    setActiveMenuGroup(activeMenuGroup ? undefined : group);
+  }
+
+  function handleMenuGroupClose() {
```

Comment:
Nitpick: I tend to put these callbacks up towards the top next to the useState stuff. 

Also I think this and handleMenuGroupToggle should be wrapped in a useCallback() to memoise it if you're passing it into another react component